daemon/network.go

```diff
@@ -801,7 +801,10 @@ func buildPeerInfoResources(peers []networkdb.PeerInfo) []networktypes.PeerInfo
 	}
 	peerInfo := make([]networktypes.PeerInfo, 0, len(peers))
 	for _, peer := range peers {
-		peerInfo = append(peerInfo, networktypes.PeerInfo(peer))
```

Comment:
the other option is to change `networkdb.PeerInfo` fields order, but I don't know the impacts
(I didn't know such a cast was valid in go!)